src/resources.rs

```diff
@@ -111,7 +117,7 @@ pub fn table_entries() -> Vec<(u32, String)> {
 fn test_table_entries() {
   let mut entries = table_entries();
   entries.sort();
-  assert_eq!(entries.len(), 3);
+  // assert_eq!(entries.len(), 3);
```

Comment:
Because a worker may be added the number of resources is no longer just the stdio....

Reply:
No longer commented out.